modules/nf-core/longphase/haplotag/main.nf

```diff
@@ -0,0 +1,66 @@
+process LONGPHASE_HAPLOTAG {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/longphase:1.7.3--hf5e1c6e_0':
+        'biocontainers/longphase:1.7.3--hf5e1c6e_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai), path(snps), path(svs), path(mods)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+
+
+    output:
+    tuple val(meta), path("*.bam") , emit: bam , optional: true
+    tuple val(meta), path("*.cram"), emit: cram, optional: true
+    tuple val(meta), path("*.log") , emit: log , optional: true
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def sv_file = params.svs ? "--sv-file ${svs}" : ""
+    def mod_file = params.mods ? "--mod-file ${mods}" : ""
+
+    """
+    longphase \\
+        haplotag \\
+        $args \\
+        --threads $task.cpus \\
+        -o ${prefix} \\
+        --reference ${fasta} \\
+        --snp-file ${snps} \\
+        --bam ${bam} \\
+        ${sv_file} \\
+        ${mod_file}
+
+    if [ -f "${prefix}.out" ]; then
+        mv ${prefix}.out ${prefix}.log
+    fi
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        longphase: \$(longphase --version | head -n 1 | sed 's/Version: //')
+    END_VERSIONS
+    """
+
+    stub:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def log = args.contains('--log') ? "touch ${prefix}.log" : ''
+    """
+    touch ${prefix}.bam
```

Comment:
Can you somehow add some logic here that will create CRAM files when the main code would? (only if it's really possible)